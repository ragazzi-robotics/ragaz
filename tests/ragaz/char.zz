def main():

    var a = 'a'
    print(isinstance(a, byte))

    var b: byte = 97
    print(isinstance(a, byte))
    print(a)

    var c: int = a as int
    print(c)

    var d = "abcd"
    d[1] = 'x'
    print(d)

    var e = d[1]
    print(isinstance(e, byte))
