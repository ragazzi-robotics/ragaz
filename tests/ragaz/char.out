True
True
a
97
axcd
True
